src/components/Navigation/NavigationContainer.js

```diff
@@ -55,6 +56,19 @@ const NavigationContainer = () => {
         return active;
     };
 
+    const isCourseFormActive = (location, NavItem) => {
+        let active = false;
+        if (location) {
+            if (
+                location.pathname.includes('course_details') &&
+                NavItem.name === 'Courses'
+            ) {
+                active = true;
+            }
+        }
+        return active;
```

Comment:
location is undefined before the jsx is rendered so this is to prevent it from freaking out. but I want to stick with this code because it's more readable than the one liner